README.md

```diff
@@ -0,0 +1,39 @@
+# Books API
+
+## Database scheme 
+```mermaid
+erDiagram
+Book{
+  long id
+  string title
+  string annotation
+  datetime createdAtUtc
+  string artworkUrl
+  int numberOfCopies
+}
+Status{
+  long id
+  string value
+}
+Author{
+  long id
+  string fullname
+}
+Language{
+  long id
+  string value
+}
+BooksAuthors{
+    long bookId
+    long authorId
+}
+BooksLanguages{
+    long bookId
+    long authorId
+}
+Book ||--|| Status: contains
+Book }|--|| BooksAuthors: bookId
+Author }|--|| BooksAuthors: authorId
+Book }|--|| BooksLanguages: bookId 
+Language }|--|| BooksLanguages: languageId
```

Comment:
It might be easier to not normalize the table here. I would suggest adding the language directly to the table with the books now.